updatecli/updatecli.d/hugo.yaml

```diff
@@ -35,50 +42,13 @@ sources:
         token: "{{ requiredEnv .github.token }}"
         username: "{{ .github.username }}"
         branch: "{{ .github.branch }}"
-  # Read netlify.toml content and update the HUGO_VERSION
-  # the content will be injected later on in the target
-  netlifyConfig:
+  updateNetlifyConfig:
     kind: file
-    name: Get netlify configuration
-    depends_on:
-      - hugoVersion
-      - hugoVersionFromNetlifyConfig
+    name: "Update Hugo version in the Netlify configuration file"
     spec:
       file: netlify.toml
-    transformers:
-      - replacer:
-          from: '{{ source "hugoVersionFromNetlifyConfig" }}'
-          to: 'HUGO_VERSION = "{{ source "hugoVersion" }}"'
-    scm:
-      github:
-        user: "{{ .github.user }}"
-        email: "{{ .github.email }}"
-        owner: "{{ .github.owner }}"
-        repository: "{{ .github.repository }}"
-        token: "{{ requiredEnv .github.token }}"
-        username: "{{ .github.username }}"
-        branch: "{{ .github.branch }}"
-conditions:
-  # Test if docker image klakegg/hugo
-  docker:
-    name: "Test if docker Image klakegg/hugoPublished is available on docker Registry"
-    sourceID: hugoVersion
-    kind: dockerImage
-    spec:
-      image: "klakegg/hugo"
-    transformers:
-      - addSuffix: "-asciidoctor"
-targets:
-  # Update Dockerfile
-  updateDockerfileArgHelmfileVersion:
-    name: "Update default HUGO_VERSION value in the Dockerfile"
-    kind: dockerfile
-    sourceID: hugoVersion
-    spec:
-      file: Dockerfile
-      instruction:
-        keyword: "ARG"
-        matcher: "HUGO_VERSION"
+      matchPattern: 'HUGO_VERSION = .*'
```

Comment:
No `-asciidoctor` suffix?

Reply:
nope: this is only the "hugo version" used in the netlify process, which rely on local binaries (and not on Docker image like the github action and local contributor's docker-compose do)